### ISSUE TITLE
Support zstd for file decompression?

### ISSUE THREAD

**Issue by romulasry** [composer/composer#11368]
See: https://github.com/kjdev/php-ext-zstd

**Comment by Seldaek** [comment]
Yeah given zstd isn't supported by github and isn't shipping by default with PHP this isn't really urgent IMO, but would be nice for sure as it does seem to be a good format.

**Comment by fabiang** [comment]
There is a plan to bundle zstd with future PHP versions: kjdev/php-ext-zstd#72

**Comment by stof** [comment]
@fabiang this issue is a question from Jordi about whether the maintainer is OK to bundle it with core. But this is not a plan yet. There hasn't been any PHP RFC submitted yet about bundling this extension.
